client/app/assets/css/superflat_redash.css

```diff
@@ -7913,6 +7913,44 @@ fieldset[disabled] .btn-inverse.active {
   position: relative;
   box-shadow: 0 1px 1px rgba(0, 0, 0, 0.07);
 }
+
+.dashboard-wrapper {
```

Comment:
Let's keep all new styles next to the components JS/HTML code.

Reply:
Sorry, I'm a bit new to the codebase, so probably chose inappropriate place for the styles. Where do you usually keep component-specific CSS?

Reply:
Most of the CSS code is still in this `superflat_redash.css` file 😞 There is #1543 where there was a try to bring in the LESS sources of the Superflat into the project, but it wasn't merged in time and now requires some rebase.

But all new CSS I try to place along side the component code, for example: https://github.com/getredash/redash/tree/master/client/app/components/app-header

Reply:
Okay, got it. If I'll have some more time - I try to review that PR and fix it. LESS is awesome :wink:

Reply:
👌 btw, instead of actually rebasing that branch I would copy over the changes to a new branch of master and then compare what was done in `app/assets/css` since the time this branch was created and apply those.